src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -221,19 +228,20 @@ private BoundExpression BindSwitchExpression(SwitchExpressionSyntax node, Bindin
                 }
                 else
                 {
-                    if (!inputType.IsErrorType())
-                        diagnostics.Add(ErrorCode.ERR_UnsupportedTypeForSlicePattern, node.Location, inputType.ToDisplayString());
                     hasErrors = true;
                     sliceType = CreateErrorType();
+                    Error(diagnostics, ErrorCode.ERR_UnsupportedTypeForSlicePattern, node, inputType);
                 }
+
+                // PROTOTYPE(list-patterns) ExternalScope?
```

Comment:
>ExternalScope [](http://example.com/codeflow?start=44&length=13)

For my own edification, what is this in reference to?

Reply:
afaik this is related to ref-like types, we should either pass the inputValEscape or check the new input, but it doesn't have coverage yet.

Reply:
Ah, right. Thanks.